strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/repository/NpmRepositoryFeatures.java

```diff
@@ -109,6 +113,23 @@ public void init()
         return defaultArtifactCoordinateValidators;
     }
 
+    public boolean allowsUnpublish(String storageId,
+                                   String repositoryId)
+    {
+        Storage storage = getConfiguration().getStorage(storageId);
+        Repository repository = storage.getRepository(repositoryId);
+
+        Optional<NpmRepositoryConfigurationData> repositoryConfiguration = Optional.ofNullable(
+                (NpmRepositoryConfigurationData) repository.getRepositoryConfiguration());
+        boolean allowsUnpublish = repositoryConfiguration.map(NpmRepositoryConfigurationData::isAllowsUnpublish)
+                                                         .orElse(ALLOWS_UNPUBLISH_DEFAULT);
+
+        logger.info("allowsUnpublish is [{}] for storageId: [{}]; repositoryId: [{}]", allowsUnpublish, storageId,
+                    repositoryId);
```

Comment:
Cosmetic: Could you please use one parameter per line, when the list is multi-line like this:
```
        logger.info("allowsUnpublish is [{}] for storageId: [{}]; repositoryId: [{}]",
                    allowsUnpublish,
                    storageId,
                    repositoryId);
```